fibonacci/fibonacci.go

```diff
@@ -0,0 +1,8 @@
+package fibonacci
+
+func fibonacci(n int) int {
+	if n <= 1 {
+		return n
+	}
```

Comment:
[performance] There is a typo in the recursive call to fibonacci(n-3) instead of fibonacci(n-2), which will cause your function to have a performance issue when computing large Fibonacci numbers.